src/main/resources/chess_board.fxml

```diff
@@ -0,0 +1,181 @@
+            <fx:reference source="square21" styleClass="dark"/>
+            <fx:reference source="square22" styleClass="light"/>
+            <fx:reference source="square23" styleClass="dark"/>
+            <fx:reference source="square24" styleClass="dark"/>
+            <fx:reference source="square25" styleClass="light"/>
+            <fx:reference source="square26" styleClass="dark"/>
+            <fx:reference source="square27" styleClass="light"/>
+            <fx:reference source="square28" styleClass="dark"/>
+            <fx:reference source="square29" styleClass="light"/>
+            <fx:reference source="square30" styleClass="dark"/>
+            <fx:reference source="square31" styleClass="light"/>
+            <fx:reference source="square32" styleClass="light"/>
+            <fx:reference source="square33" styleClass="dark"/>
+            <fx:reference source="square34" styleClass="light"/>
+            <fx:reference source="square35" styleClass="dark"/>
+            <fx:reference source="square36" styleClass="light"/>
+            <fx:reference source="square37" styleClass="dark"/>
+            <fx:reference source="square38" styleClass="light"/>
+            <fx:reference source="square39" styleClass="dark"/>
+            <fx:reference source="square40" styleClass="dark"/>
+            <fx:reference source="square41" styleClass="light"/>
+            <fx:reference source="square42" styleClass="dark"/>
+            <fx:reference source="square43" styleClass="light"/>
+            <fx:reference source="square44" styleClass="dark"/>
+            <fx:reference source="square45" styleClass="light"/>
+            <fx:reference source="square46" styleClass="dark"/>
+            <fx:reference source="square47" styleClass="light"/>
+            <fx:reference source="square48" styleClass="light"/>
+            <fx:reference source="square49" styleClass="dark"/>
+            <fx:reference source="square50" styleClass="light"/>
+            <fx:reference source="square51" styleClass="dark"/>
+            <fx:reference source="square52" styleClass="light"/>
+            <fx:reference source="square53" styleClass="dark"/>
+            <fx:reference source="square54" styleClass="light"/>
+            <fx:reference source="square55" styleClass="dark"/>
+            <fx:reference source="square56" styleClass="dark"/>
+            <fx:reference source="square57" styleClass="light"/>
+            <fx:reference source="square58" styleClass="dark"/>
+            <fx:reference source="square59" styleClass="light"/>
+            <fx:reference source="square60" styleClass="dark"/>
+            <fx:reference source="square61" styleClass="light"/>
+            <fx:reference source="square62" styleClass="dark"/>
+            <fx:reference source="square63" styleClass="light"/>
+        </ArrayList>
+    </fx:define>
+
+    <TableView fx:id="tableView" GridPane.columnIndex="9" GridPane.rowIndex="1"
```

Comment:
Same here.
When we do this it will be someone else's responsibility to create it.